apiserver/userservice/userimpl.go

```diff
@@ -80,9 +80,15 @@ func UpdateUser(request *msgs.UpdateUserRequest, pgouser string) msgs.UpdateUser
 
 	getDefaults()
 
-	if request.Username == "" {
+	if request.Username == "" && request.Expired == "" {
+		resp.Status.Code = msgs.Error
+		resp.Status.Msg = "Either --expired or --username must be set. See 'pgo update user -h' for usage."
+		return resp
+	}
+
+	if request.Username != "" && request.Expired != "" {
 		resp.Status.Code = msgs.Error
-		resp.Status.Msg = "--username is required flag"
+		resp.Status.Msg = "The --expired flag executes against all expiring users. See 'pgo update user -h' for usage."
```

Comment:
This needs to be more along the lines of "The --expired flag cannot be used when a user is specified"  and the help and docs both indicate that. 

Also any adverse effect if --all and --expired are used - any logic that gets executed that we don't want to in that case?